webcommon/cordova/src/org/netbeans/modules/cordova/CordovaPerformer.java

```diff
@@ -324,15 +325,16 @@ private Properties properties(Project p) {
         props.put(PROP_ANDROID_PROJECT_ACTIVITY, activity);//NOI18N
         
         MobilePlatform iosPlatform = PlatformManager.getPlatform(PlatformManager.IOS_TYPE);
+        if (iosPlatform != null) {
+            final String provisioningProfilePath = iosPlatform.getProvisioningProfilePath();
+            if (provisioningProfilePath != null) {
+                props.put(PROP_PROVISIONING_PROFILE, provisioningProfilePath);
+            }
+            final String codeSignIdentity = iosPlatform.getCodeSignIdentity();
 
-        final String provisioningProfilePath = iosPlatform.getProvisioningProfilePath();
-        if (provisioningProfilePath != null) {
-            props.put(PROP_PROVISIONING_PROFILE, provisioningProfilePath);
-        }
-        final String codeSignIdentity = iosPlatform.getCodeSignIdentity();
-
-        if (codeSignIdentity != null) {
-            props.put(PROP_CERTIFICATE_NAME, codeSignIdentity);
+            if (codeSignIdentity != null) {
+                props.put(PROP_CERTIFICATE_NAME, codeSignIdentity);
+            }
         }
 
         //workaround for some strange behavior of ant execution in netbeans
```

Comment:
there's also a reference to `iosPlatform` at line 357; is it OK to leave unchecked ?